native/src/modules/navigation/__tests__/navigateToDeepLink.spec.js

```diff
@@ -440,6 +457,7 @@ describe('navigateToDeepLink', () => {
         undefined,
         false
       )
+      expect(sendTrackingSignal).toHaveBeenCalledTimes(1)
```

Comment:
Can you add a test to check if sendTrackingSignal is called with correct parameters?

Reply:
Thanks for the feedback, i added checks for that.